src/components/organisms/footer/footer.scss

```
@@ -0,0 +1,84 @@
+@use "../../variables" as *;
+
+.footer {
+  text-align: center;
+  align-items: center;
+  color: $black;
+  font-size: $fontsize-s;
+
+  .contact {
+    border-bottom: 1px solid $gray_nobel;
+    padding: 52px 20px;
+    @media screen and(max-width:$breakpoint-md) {
+      padding: 32px 20px;
+    }
+    .wrapper {
+      padding-bottom: 20px;
+      img {
+        width: 150px;
+      }
+      adress {
+        line-height: 28px;
+      }
+    }
+    .icon-list {
+      display: flex;
+      justify-content: center;
+      gap: 14px;
+      .icon {
+        color: $black;
+
+        i {
+          width: 14px;
+          height: 14px;
+          transition: color 0.2s ease;
+          &:hover {
+            color: $gray_nobel;
+          }
+        }
+      }
+    }
+  }
+  nav {
+    padding: 16px;
+    background-color: $white_smoke;
+    ul {
+      margin-bottom: 0px;
+      padding: 0px;
+      list-style: none;
+      columns: 8;
+      column-rule: 1px solid;
+      column-gap: 8px;
```

```suggestion
      display: flex;
      justify-content: center;
```
columnを使うとそれぞれのカラムが等幅になってしまうため、現在の書き方ではmd以下の時のFigmaのデザインを再現することができません。
columnのほうがスマートにかけるのは確かですが、横並びのレイアウトはflexboxで実装しましょう。